app/src/main/kotlin/co/netguru/baby/monitor/client/feature/machinelearning/WavFileGenerator.kt

```diff
@@ -20,18 +20,18 @@ object WavFileGenerator {
     private const val AVAILABLE_SPACE = AVAILABLE_MEGABYTES_FOR_APPLICATION * BYTES_IN_MEGABYTE
 
     fun saveAudio(
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/d34b94ec1c855c6823fe80604ac7e485f795cffa/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/machinelearning/WavFileGenerator.kt#L22"></span>The function saveAudio has too many parameters. The current threshold is set to 6.</td>
    </tr>
  </tbody>
</table>